T100TA_guide.md

```diff
@@ -168,12 +168,14 @@ mmcblk2boot1
 mmcblk0
 └─mmcblk0p1  vfat                 9016-4EF8             /media/xubuntu/9016-4EF8
 ```
+- If your ext4 partition is not mounted, you must do it
+  * `mount /dev/mmcblk2p2 /target`
 - Mount the EFI System Partition on the new system
   * `mount /dev/mmcblk2p1 /target/boot/efi`
 - Then, we have to mount some other filesystems before chrooting:
 ```
 for dir in /dev /dev/pts /proc /run /sys;
-  do mount --bind "$dir" /target/"$dir";
+  do mount --bind "$dir" /target"$dir";
```

Comment:
You are right, this slash is useless for the program to work properly. It is just there for readability.

Reply:
I agree, with the slash we understand we have to mount all directories as /target subdirectoires, but the command does not work.